docs/documentation/reviewing-documentation.md

```diff
@@ -18,7 +19,18 @@ Content reviews are for validating the correctness of technical steps and detail
 - Steps that may be particularly conducive to software or default configuration as a solution
 - Lack of clarity or any other confusion you may have
 
-Editorial Review
-----------------
+Additionally, ensure that the document meets our [style](/documentation/style-guide) and
+[layout](/documentation/writing-documentation#document-layout) guidelines, as well as correct spelling and grammar.
 
-Editorial reviews are for ensuring docs meet our [style](/documentation/style-guide) and [layout](/documentation/writing-documentation#document-layout) guidelines; improving readability; and proofing spelling, grammar, and punctuation.
+Completing a Review
+-------------------
+
+Upon completion of the review:
+
+1.  Update (or add) the review date in the
+    [metadata](https://squidfunk.github.io/mkdocs-material/extensions/metadata/#redirecting-to-another-page) at the top
```

Comment:
I didn't find this document to be helpful so IMO just skip the link.

What does the finished product look like?